src/components/ConfigEditor.tsx

```diff
@@ -59,6 +59,7 @@ export function ConfigEditor(props: Props) {
             value={jsonData.apiKey || ''}
             placeholder="Analytics API Key"
             width={40}
+            id={`config-editor-${props.options.name}_api-key-input`}
```

Comment:
I would still give it another 30mins to investigate, if grafana UI components like here `<Input>` do not support some other attribute for test selectors, because `id` is special and has to be unique, so it would be better to have some `data-testid` / `data-playwrightid` or something else 
I know that `data-testid` is not valid prop and is not present in DOM , but maybe upgrade of grafana UI library, or some other "test attribute" could work ? I would still give it another try

Reply:
I investigated it again and found out that for all elements beside the `<Select>` and `<RadioButtonGroup>` the `data-testid` is available. I didn't see it before as I was writing the first test for the `<Select>` elements and then just assumed that the other elements also wouldn't work. So with the last commit I changed all `id`s to `data-testid`s for all elements except the `<Select>` and `<RadioButtonGroup>`.
Tried to find a way to also use `data-testid` for the other elements, but nothing I tried worked. Not entirely sure but I think it works for the other elements because under the hood they are simple HTML elements and inherit HTMLElement Attributes, whereas the other two elements (`<Select>` and `<RadioButtonGroup>`) are React JSX elements.